### PR TITLE
ConversationAPI quick guide

### PR DESCRIPTION
**Pull request by matokeni**

### CODE REVIEW

File: docs/conversation/conversation_quick_guide.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Quick Guide
```
Comment (rumen-kyusakov) [inline docs/conversation/conversation_quick_guide.md:2]: `Quickstart Guide` or `Quick Start Guide`

File: docs/conversation/conversation_quick_guide.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## 2. Selecting and adding channels
+
+Once step 1 is completed, start setting up the [**channels**](doc:conversation#supported-channels) you want to include in the conversation App.
+
+Select the channel you wish to set up. Keep in mind that you need to register each channel before you can include it in a conversation App.
+
+![New Channel](images/dashboard/dashboard_add_channels.png)
+
+Depending on the channel, select your ID/Identity to configure your channel to work with your Conversation API App. Note that you must first set up a Service plan ID/Service ID/Sender Identity at corresponding channel portals.
```
Comment (rumen-kyusakov) [inline docs/conversation/conversation_quick_guide.md:28]: `your ID/Identity` -> `channel sender identity`

File: docs/conversation/conversation_quick_guide.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+After choosing your ID/identity, the underlying channel needs to update the callback URL which will automatically overwrite and update any current callback URL setup.
```
Comment (rumen-kyusakov) [inline docs/conversation/conversation_quick_guide.md:30]: your ID/Identity -> channel sender identity
Also maybe slightly better to say: the callback URL of the underlying channel should be updated to point to the channel adapter of the Conversation API. Note: the callback URL for most channels will be updated automatically which will overwrite the current callback URL.

File: docs/conversation/conversation_quick_guide.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+hidden: false
+---
+
+In this guide we’ll go through the necessary steps of creating your first App.
+
+## 1. Creating and configuring a new App
+
+Go to the [Apps page](https://dashboard.sinch.com/convapi/apps) to get started on creating your App.
+ 
+Once you’ve started to create a new App you are prompted to specify an internal display name as well as in which [**region**](doc:conversation#regions) the App should be stored.
```
Comment (Nerja) [inline docs/conversation/conversation_quick_guide.md:14]: Add: "Note that it is not possible to migrate the app to another region"?

File: docs/conversation/conversation_quick_guide.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+**Access keys** are required for authentication purposes when using the Conversation API.
+
+You can generate an access key under Settings -> Access Keys page by clicking on *"New Key"*.
+
+![Access Keys](images/dashboard/dashboard_access_keys.png)
+
+A **client_id** and **client_secret** will be provided when creating an Access Key in the portal. The secret is only shown after generation, copy and store it in a safe place. 
+
+A generated access key can be used in conjunction with your project ID to interact with the Conversation API. Read more about possible authentication methods at [**Authentication**](doc:conversation#authentication).
```
Comment (rumen-kyusakov) [inline docs/conversation/conversation_quick_guide.md:70]: I think you should mention that you get an access token given the key id and key secret using a region specific token service at https://<region>.auth.sinch.com/oauth2/token. The access token is then used in conjunction with your project ID to interact with the Conversation API.

File: docs/conversation/conversation_quick_guide.md (hunk 4 above)
Comment (matokeni) [reply docs/conversation/conversation_quick_guide.md:14]: yep, added!

File: docs/conversation/conversation_quick_guide.md (hunk 1 above)
Comment (matokeni) [reply docs/conversation/conversation_quick_guide.md]: 👌

File: docs/conversation/conversation_quick_guide.md (hunk 2 above)
Comment (matokeni) [reply docs/conversation/conversation_quick_guide.md]: 👌

File: docs/conversation/conversation_quick_guide.md (hunk 3 above)
Comment (matokeni) [reply docs/conversation/conversation_quick_guide.md]: ah nice. I was trying to rephrase but this is much better 👍

File: docs/conversation/conversation_quick_guide.md (hunk 5 above)
Comment (matokeni) [reply docs/conversation/conversation_quick_guide.md]: yep, added a section about that!

**Pull request merged**